src/components/modal/Modal.scss

```diff
@@ -144,8 +144,23 @@
             padding: 0;
         }
 
+        .modal-header-container {
+            position: fixed;
```

Comment:
Since `.modal-dialog` has `position: relative;`, could we utilize that and use `absolute` positioning?

Not using `fixed` would be ideal if possible.

Reply:
Found a way to use `position: absolute` instead.